prime-router/src/test/kotlin/cli/ValidateSettingCommandsTest.kt

```diff
@@ -0,0 +1,134 @@
+package gov.cdc.prime.router.cli
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class ValidateSettingCommandsTest {
```

Comment:
Clikt exposes a `test` function that should be used for testing commands.

Reply:
> Clikt exposes a `test` function that should be used for testing commands.

test target implemented covering validation settings with json schemas